samples/jwt-server/jwt-server.yaml

```diff
@@ -25,9 +25,106 @@ spec:
   - name: http
     port: 8000
     targetPort: 8000
+  - name: https
+    port: 8443
+    targetPort: 8443
   selector:
     app: jwt-server
 ---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: jwt-server-cert-key
+data:
+  #command to generate certificate and a key
```

Comment:
are current certs sufficient? like `tests/testdata/certs/cert.crt`?

Not a big concern, but it gets confusing to have a bunch of test certs around.

Reply:
I agree but I would require SAN enabled certs and for this "tests/testdata/certs/cert.crt", SAN is empty.